File: Php/Thief.php
<?php 
require_once ("loadingClass.php");

class Thief extends Caracter{

    public function __construct ($pseudo, $age) {
        parent::__construct($pseudo, $age);
        $this -> setRace ("Thief");
    }

    public function fight () {
        if ($this -> weapons = "Dagger") {
            echo "</br> I can use this Dagger";
        }
    }

}


?>

File: Java POO/Thief.java
package game;

 class Thief extends Person {
		
	public Thief (String pseudo, int age) {
		super (pseudo, age);
		this.setRace("Thief");
	}	
	
	public void fight () {
		if (this.weapons.equals("Dagger")) {
			System.out.println("I can use that");
		} 
	}
}
